packages/config/config/devices/0x007a/506219.json

```diff
@@ -39,7 +39,7 @@
 	},
 	"paramInformation": {
 		"0": {
-			"label": "Switching/dimming/shutters dual-surface:",
+			"label": "Switching/dimming/shutters dual-surface: ",
```

Comment:
The old label had the same flaw, but we should make sure that there are no trailing punctuation or whitespaces in labels